flake.nix

```diff
@@ -113,7 +119,12 @@
 
       nixosModules.ogmios = { pkgs, ... }: {
         imports = [ ./nix/ogmios-nixos-module.nix ];
-        nixpkgs.overlays = [ (_: _: { ogmios = self.flake.${pkgs.system}.packages."ogmios:exe:ogmios"; }) ];
+        nixpkgs.overlays = [
+          (_: _: {
+            ogmios = self.flake.${pkgs.system}.packages."ogmios:exe:ogmios";
+            inherit (inputs) cardano-configurations;
```

Comment:
I think that this the first time I see a non derivation in an overlay. Why don't pass it as extra argument to the modules? Perhaps some tool/framework assumes that everything here is a derivation. Just like when you use `nix flake check` with a flake which `packages` are not derivations.

Reply:
it's a derivation. non-flake input.